website/docs/components/application-state/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,7 @@
+## Usage
+
+### When to use
+
+Some text here.
```

Comment:
Would you want to wait until this and other docs content is added before merging to main? (Since this PR has been marked as ready for review.)

Reply:
I would merge #1264 before merging this one. Would you prefer that I move this back into a draft?

Reply:
I would probably move it back to draft until it's ready to be merged all together. (I wasn't sure if you intended that it be merged first and that those other sections of the doc would be completed afterwards for instance.)

Reply:
ok, converted back to draft.